components/library/msgpack/Makefile

```diff
@@ -10,25 +10,26 @@
 
 #
 # Copyright 2022-2023 Niklas Poslovski
+# Copyright 2025 Klaus Ziegler
 #
 
 BUILD_STYLE=cmake
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=         msgpack
-COMPONENT_VERSION=      6.0.0
-#COMPONENT_REVISION=     1
+COMPONENT_VERSION=      7.0.0
 COMPONENT_PROJECT_URL=  https://msgpack.org
 COMPONENT_SUMMARY=      An efficient object serialization library
 COMPONENT_FMRI=         library/msgpack
-COMPONENT_SRC=          $(COMPONENT_NAME)-c-$(COMPONENT_VERSION)
+COMPONENT_SRC=          $(COMPONENT_NAME)-cxx-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_HASH= \
-    sha256:3654f5e2c652dc52e0a993e270bb57d5702b262703f03771c152bba51602aeba
-COMPONENT_ARCHIVE_URL=	https://github.com/msgpack/msgpack-c/releases/download/c-$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+    sha256:7504b7af7e7b9002ce529d4f941e1b7fb1fb435768780ce7da4abaac79bb156f
+COMPONENT_ARCHIVE_URL=	https://github.com/msgpack/msgpack-c/releases/download/cpp-$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
 COMPONENT_CLASSIFICATION=       System/Libraries
 COMPONENT_LICENSE=      Boost
 COMPONENT_LICENSE_FILE= LICENSE_1_0.txt
+COMPONENT_BUILD_ARGS= -j50
```

Comment:
Please _never_ do this.

If you need parallel cmake build then use `USE_PARALLEL_BUILD = yes` instead.  If the default parallelism is not okay for you, then build using something like `env - gmake PARALLEL_JOBS=50 publish`.